tuf/ngclient/fetcher.py

```diff
@@ -35,9 +35,9 @@ def fetch(self, url: str) -> Iterator[bytes]:
             url: A URL string that represents a file location.
 
         Raises:
-            tuf.exceptions.SlowRetrievalError: A timeout occurs while receiving
```

Comment:
(this is more a generic comment)
I noticed that we refer to the full exception path; in other docstrings  (including tuf/api/metadata), we use only the exception name. Not sure which should be the standard.

Reply:
My opinion is that we shouldn't use the full exception path as it can change (as it happened now).
@jku what do you think?

Reply:
I personally think including the "module path" at all is not very useful in docstrings: it just makes things harder to read. But if we do include them, then they should be complete